docs/create-org.rst

```diff
@@ -5,12 +5,13 @@ Create a new organization
 =========================
 
 This is a guide on how to create a new organization in the CrateDB Cloud
-Console. If you follow the :ref:`cluster deployment tutorial
-<cloud-tutorials:cluster-deployment>`, you will create an
-organization as part of the signup process via the Console wizard. Currently,
-clusters can only be deployed by this method. Should you, however, access the
-Console without following this method, or have deleted your organization, this
-guide will inform you on how to create one from scratch.
+Console. If you follow the `cluster deployment tutorial`_, you will create an
+organization as part of the signup process. There are scenarios where you may
+want to create a new organization, however. If you arrive at the Console for
+the first time, or if you have deleted your last organization, you will have
```

Comment:
Not sure if this is the intended meaning but I don't think it's possible to delete an organization if it's your only organization. I don't think this was specified in any A/Cs but from what I can tell on dev, you can't delete the last organization you belong to.

Reply:
Really? I tested on production with a Github account and it works just fine. Can create an org, delete it again, create it again, ad infinitum.